sql/1.107/contributor_activity.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE contributor_activity (
+    id serial NOT NULL PRIMARY KEY,
+    github_id text NOT NULL UNIQUE,
```

Comment:
What is the difference between `github_id` and `contributor_id`?

We probably don't want to delete events if the user or repo goes away.  Events still happened that we may want to evaluate.